runtime/centrifuge/src/lib.rs

```diff
@@ -1506,16 +1505,16 @@ impl pallet_keystore::pallet::Config for Runtime {
 }
 
 parameter_types! {
-	// per byte deposit is 0.01 AIR
-	pub const DepositPerByte: Balance = CENTI_CFG;
-	// Base deposit to add attribute is 0.1 AIR
-	pub const AttributeDepositBase: Balance = 10 * CENTI_CFG;
-	// Base deposit to add metadata is 0.1 AIR
-	pub const MetadataDepositBase: Balance = 10 * CENTI_CFG;
-	// Deposit to create a class is 1 AIR
-	pub const CollectionDeposit: Balance = CFG;
-	// Deposit to create a class is 0.1 AIR
-	pub const ItemDeposit: Balance = 10 * CENTI_CFG;
+	// per byte deposit is 0.1 CFG
+	pub const DepositPerByte: Balance = 10 * CENTI_CFG;
+	// Base deposit to add attribute is 1 CFG
+	pub const AttributeDepositBase: Balance = 1 * CFG;
+	// Base deposit to add metadata is 1 CFG
+	pub const MetadataDepositBase: Balance = 1 * CFG;
+	// Deposit to create a class is 10 CFG
+	pub const CollectionDeposit: Balance = 10 * CFG;
+	// Deposit to create a class is 1 CFG
+	pub const ItemDeposit: Balance = 1 * CFG;
```

Comment:
This means that to create a pool with 1000 loans, the deposit is 1000 CFG (per item) + 3000 CFG (NFTs minted by the POD have 2 attributes and 1 metadata field) + metadata size based deposit. That's fine now but can get expensive later, and this disincentivizes on-chain transparency (cheaper to create 1 asset NFT with many bundled loans, than 1000 individual asset NFTs). I would rather keep all of these a bit lower, potentially just returning to what they were. I would probably increase the collection deposit in return, e.g. to 100 CFG.